src/components/Reports/Reports.js

```diff
@@ -17,17 +17,21 @@ import { addDays } from '../../utils';
 
 const periodSelectedDaysDefault = 7;
 
+const getCurrentDate = (date) => {
```

Comment:
In this case, _current_ is wrong, because you are getting the _pure date_ or _begining of the date_ passed as parameter. Let me research for common names for this operation.

Reply:
In _Moment.js_ they use _[startOf](https://momentjs.com/docs/#/manipulating/start-of/)_, what do you think about using the same convention than they?

```suggestion
const getStartOfDate = (date) => {
```

Reply:
changed